NEWS.md

```diff
@@ -1,6 +1,10 @@
 # EpiNow2 1.2.0
+
 * Added timeout and timing option to `regional_epinow`
 * Improved logging of warnings in `regional_epinow`
+* Added inner 20% high density region.
```

Comment:
I can see we don't have a 1.2.0 tag officially published yet but we are using 1.2 in covid-rt-estimates. We should probably have a chat on a weekly catchup about how we are going to manage releases as it's an easy way to get in a sticky mess with multiple projects and dependencies.

Reply:
i've been publishing tags when a version is wrapped up but happy to discuss. Sounds like a good idea!